addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -333,6 +302,71 @@
                 </div>
             </div>
         </t>
+        <t t-elif="channel.enroll in ['public', 'invite']" name="o_wslides_prerequisite_test">
+            <div t-if="channel.prerequisite_channel_id and not channel.has_completed_prerequisite"
```

Comment:
shouldn' we also check that in `_action_add_members` ? (currently it's possible to join the course with a RPC call)

if you do it you can add a small test

Reply:
Actually no, it's purely informative. Anyone can join a course with a prerequisite without finishing it first :smiley: